src/pipelines/hospitalizations/gb_authority.py

```diff
@@ -56,3 +57,43 @@ def parse(self, sources: Dict[str, str], aux: Dict[str, DataFrame], **parse_opts
         )
 
         return hospitalized.merge(intensive_care, how="outer")
+
+
+class UKL1DataSource(DataSource):
+    def parse_dataframes(
+        self, dataframes: Dict[str, DataFrame], aux: Dict[str, DataFrame], **parse_opts
+    ) -> DataFrame:
+
+        # Specify filter for overview / consolidated data
+        # for the UK
+        api_filter_overview = [
+            "areaType=overview"
+        ]
+
+        # Specify relevant metrics that will be used
+        # according to Google's schema
+        api_structure_hospitalization = {
+            "date": "date",
+            "newAdmissions": "newAdmissions",
+            "cumAdmissions": "cumAdmissions",
+            "hospitalCases": "hospitalCases",
+            "covidOccupiedMVBeds": "covidOccupiedMVBeds"
+        }
+
+        api = Cov19API(filters=api_filter_overview,
+                       structure=api_structure_hospitalization)
+
+        data = api.get_dataframe()
+
+        # Rename columns and map to expected schema
+        data = data.rename(columns={
```

Comment:
We have a function table_rename that wraps this - can you use that instead for consistency?

Reply:
Done!